build/run-backup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+XBCLOUD_ARGS="--curl-retriable-errors=7 --parallel=10"
+MD5_ARG="--md5"
+if [ -n "$VERIFY_TLS" ] && [[ $VERIFY_TLS == "false" ]]; then
+	INSECURE_ARG="--insecure"
+fi
+
+request_backup() {
+	curl -s http://${SRC_NODE}:6033/backup/${BACKUP_NAME} -o /backup/${BACKUP_NAME}.stream
+}
+
+request_logs() {
+	curl -s http://${SRC_NODE}:6033/logs/${BACKUP_NAME}
+}
+
+full_backup_name() {
+	echo "${BACKUP_NAME}-$(date +%Y-%m-%dT%H:%M:%S)-full"
+}
+
+run_s3() {
+	cat /backup/${BACKUP_NAME}.stream \
+		| xbcloud put ${XBCLOUD_ARGS} ${MD5_ARG} ${INSECURE_ARG} "$(full_backup_name)" --storage=s3 --s3-bucket="${S3_BUCKET}"
+}
+
+run_gcs() {
+	cat /backup/${BACKUP_NAME}.stream \
+		| xbcloud put ${XBCLOUD_ARGS} ${MD5_ARG} ${INSECURE_ARG} "$(full_backup_name)" --storage=google --google-bucket="${GCS_BUCKET}"
+}
+
+run_azure() {
+	cat /backup/${BACKUP_NAME}.stream | xbcloud put ${XBCLOUD_ARGS} ${INSECURE_ARG} "$(full_backup_name)" --storage=azure
+}
+
+main() {
+	echo "Running backup ${BACKUP_NAME} on ${SRC_NODE}"
+
+	if [ ! -f /backup/${BACKUP_NAME}.stream ]; then
+		request_backup
+
+		echo "xtrabackup logs:"
+		request_logs
+	fi
+
+	case ${STORAGE_TYPE} in
+		"s3") run_s3 ;;
+		"gcs") run_gcs ;;
+		"azure") run_azure ;;
+	esac
+
+	echo "Backup finished and uploaded successfully: $(full_backup_name)"
```

Comment:
Message in the backup logs is not correct, one directory is created, but another one is mentioned in the backup uploaded message:
```
220610 09:59:30 xbcloud: successfully uploaded chunk: backup1-2022-06-10T09:59:19-full/xtrabackup_info.00000000000000000001, size: 29
220610 09:59:30 xbcloud: successfully uploaded chunk: backup1-2022-06-10T09:59:19-full/xtrabackup_tablespaces.00000000000000000000, size: 95
220610 09:59:30 xbcloud: successfully uploaded chunk: backup1-2022-06-10T09:59:19-full/xtrabackup_tablespaces.00000000000000000001, size: 36
220610 09:59:31 xbcloud: Uploading md5
220610 09:59:31 xbcloud: Upload completed.
Backup finished and uploaded successfully: backup1-2022-06-10T09:59:31-full
```
`backup1-2022-06-10T09:59:19-full` vs `backup1-2022-06-10T09:59:31-full`

Reply:
fixed